permission/bind/nodes.go

```diff
@@ -31,7 +31,7 @@ var (
0016040519081016040528093929190818152602001838380828437600081840152601f19601f82011690508083019250505050505050815260200185858080601f016020809104026020016040519081016040528093929190818152602001838380828437600081840152601f19601f8201169050808301925050505050505081526020016002815250908060018154018082558091505090600182039060005260206000209060030201600090919290919091506000820151816000019080519060200190612d3d9291906130fa565b506020820151816001019080519060200190612d5a9291906130fa565b50604082015181600201555050507f0413ce00d5de406d9939003416263a7530eaeb13f9d281c8baeba1601def960d858585856040518080602001806020018381038352878782818152602001925080828437600081840152601f19601f8201169050808301925050508381038252858582818152602001925080828437600081840152601f19601f820116905080830192505050965050505050505060405180910390a15050505050565b6000816040516020018080602001828103825283818151815260200191508051906020019080838360005b83811015612e4c578082015181840152602081019050612e31565b50505050905090810190601f168015612e795780820380516001836020036101000a031916815260200191505b5092505050604051602081830303815290604052805190602001206001612e9f8561304f565b815481101515612eab57fe5b90600052602060002090600302016001016040516020018080602001828103825283818154600181600116156101000203166002900481526020019150805460018160011615610100020316600290048015612f485780601f10612f1d57610100808354040283529160200191612f48565b820191906000526020600020905b815481529060010190602001808311612f2b57829003601f168201915b5050925050506040516020818303038152906040528051906020012014905092915050565b60008060026000846040516020018080602001828103825283818151815260200191508051906020019080838360005b83811015612fb8578082015181840152602081019050612f9d565b50505050905090810190601f168015612fe55780820380516001836020036101000a031916815260200191505b509250505060405160208183030381529060405280519060200120815260200190815260200160002054141561301e576000905061304a565b60016130298361304f565b81548110151561303557fe5b90600052602060002090600302016002015490505b919050565b6000600160026000846040516020018080602001828103825283818151815260200191508051906020019080838360005b8381101561309b578082015181840152602081019050613080565b50505050905090810190601f1680156130c85780820380516001836020036101000a031916815260200191505b509250505060405160208183030381529060405280519060200120815260200190815260200160002054039050919050565b828054600181600116156101000203166002900490600052602060002090601f016020900481019282601f1061313b57805160ff1916838001178555613169565b82800160010185558215613169579182015b8281111561316857825182559160200191906001019061314d565b5b509050613176919061317a565b5090565b61319c91905b80821115613198576000816000905550600101613180565b5090565b9056fe656e6f646520696420646f6573206e6f742062656c6f6e6720746f2074686520706173736564206f7267696e76616c6964206f7065726174696f6e2e2077726f6e6720616374696f6e20706173736564656e6f646520696420646f6573206e6f742062656c6f6e6720746f2074686520706173736564206f7267206964a165627a7a72305820a46616495a957989964cf36c1721df3dca4c5c0e613c53dd53eb454f4c07972a0029`
```

Comment:
BIN looks the same, why are we updating this?

Reply:
as I mentioned yesterday, did a regeneration of all binding code as it was not done for sometime